spaces/domains.json

```
@@ -7,6 +7,7 @@
   "vote.kargain.world": "kargainworld.eth",
   "vote.lobis.finance": "lobis.eth",
   "vote.euphoria.money": "euphoriadao.eth",
+  "vote.nodegamexyz.com: "nodegame.eth",
```

You will miss a quote here.
```suggestion
  "vote.nodegamexyz.com": "nodegame.eth",
```